crates/bin/pd/src/migrate/testnet78.rs

```diff
@@ -536,8 +540,34 @@ fn truncate(s: &str, max_bytes: usize) -> &str {
     &s[..closest]
 }
 
-mod tests {
+/// Edit the node's CometBFT config file to set `mempool.max_txs_bytes` to the required value.
+/// This value will affect consensus, but the config setting is specified for CometBFT
+/// specifically.
+#[instrument]
+pub(crate) fn update_cometbft_mempool_settings(cometbft_home: PathBuf) -> anyhow::Result<()> {
+    let cometbft_config_path = cometbft_home.join("config").join("config.toml");
+    tracing::debug!(cometbft_config_path = %cometbft_config_path.display(), "opening cometbft config file");
+    let mut cometbft_config = TendermintConfig::load_toml_file(&cometbft_config_path)
+        .context("failed to load pre-migration cometbft config file")?;
+    // We expect the pre-migration value to be '1073741824', assuming it's unedited from `pd testnet join`.
+    // let pre_migration_value = cometbft_config.mempool.max_txs_bytes;
+    // The new value was updated in GH4594 and represents ~10MB or (2^20 * 10).
+    let post_migration_value = 10485760;
```

Comment:
There are two `max_tx_bytes` parameters that we want to change:
- the first one is a `mempool` config parameter that controls the maximum size of a transaction to be included in the mempool/gossiped to other nodes 
- the second is a consensus parameter that controls the maximum size of block data (including headers)

We want to change both values:
- the mempool value should be set to something around 30KB (x-ref #4629)
- the consensus value should be set to 1MIB (x-ref #block size #4597)

Reply:
There are two different mempool configuration parameters that are easily mixed up.

- `max_txs_bytes` is what this PR is about
- `max_tx_bytes` is what #4629 is about, though it'd make sense to add it to this PR, too, after #4632 gets merged.

Reply:
The `max_tx_bytes` vs `max_txs_bytes` distinction is certainly ripe for confusion, but if we're changing two values in the CometBFT config for Testnet 78, then we should update both values as part of the migration flow. I'll make a change to this PR to accommodate.